server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
> and will only know that it broke via FF desktop

The way I see this being implemented in desktop is that we will query this end-point at login time and cache the values in prefs. So for an account that remains logged in, we'd do this exactly once.

This means we'd only notice the breakage for new Sync signins, so may go unnoticed for a longer time than if it broke already configured accounts.

> it wouldn't take much to talk me out of including the sync URL here

OTOH, if that's difficult to maintain here, it would probably be even more difficult to maintain in https://dxr.mozilla.org/mozilla-central/rev/63cc31d6cc1c8089590461016ce0b4a2fb77ecbc/browser/app/profile/firefox.js#1278, and even more difficult to change once we notice it broken.